extensions/kafka-eight-simpleConsumer/src/main/resources/META-INF/services/io.druid.initialization.DruidModule

```diff
@@ -0,0 +1 @@
+io.druid.firehose.kafka.KafkaEightSimpleConsumerDruidModule
```

Comment:
my intellij always adds them, but if they are not general convention then no big deal. all files should still be covered by root license right?